weblate/trans/views/create.py

```diff
@@ -521,6 +521,9 @@ def form_valid(self, form):
                 slug=form.cleaned_data["slug"],
                 vcs=component.vcs,
                 source_language=component.source_language.pk,
+                license=component.license,
+                agreement=component.agreement,
+                commit_message=component.commit_message,
```

Comment:
Probably category should be kept as well. And there are more message configurations -            "add_message",             "delete_message",              "merge_message",             "addon_message",             "pull_message".